riscv/riscv.mk.in

```diff
@@ -1315,26 +1315,29 @@ riscv_insn_ext_zicond = \
         czero_eqz \
         czero_nez \
 
+# Note that riscv_insn_ext_p and riscv_insn_ext_zvk contain instructions
+# that have conflicting encodings. They cannot be both included concurrently.
+# TODO: have a mechanism to enable those groups at runtime.
```

Comment:
I see you added this comment here and removed it in the next PR.  Can you instead delete the comment here so it doesn't appear in the history?

Reply:
Done, thanks for catching this.